.travis.yml

```diff
@@ -19,6 +24,7 @@ deploy:
     skip_cleanup: true
     script: bash $TRAVIS_BUILD_DIR/.travis.d/deploy_docs.sh
     on:
+        repo: vitasdk/vita-headers
```

Comment:
Doesn't it prevent non-vitasdk members from easily enabling documentation deployment on their fork so as to test it?